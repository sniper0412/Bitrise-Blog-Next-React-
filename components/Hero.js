import React from 'react';

import Subscription from './Subscription';

const Hero = () => (
  <div className="hero">
    <div className="content-wrapper">
      <div className="hero-inner">
        <div className="newsletter">
          <h1>Bitrise Blog</h1>
          <div className="text">
            <p>
              Articles about Continuous Integration & Delivery, automation, iOS code signing, DevOps, and awesome tech
            </p>
            <Subscription />
          </div>
        </div>
      </div>
    </div>
  </div>
);

Hero.propTypes = {};

export default Hero;
