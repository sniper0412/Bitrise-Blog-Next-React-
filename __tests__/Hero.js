import renderer from 'react-test-renderer';

import Hero from '../components/Hero';

describe('Hero component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Hero />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
