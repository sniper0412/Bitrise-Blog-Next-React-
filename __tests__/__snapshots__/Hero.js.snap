// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component renders correctly 1`] = `
<div
  className="hero"
>
  <div
    className="content-wrapper"
  >
    <div
      className="hero-inner"
    >
      <div
        className="newsletter"
      >
        <h1>
          Bitrise Blog
        </h1>
        <div
          className="text"
        >
          <p>
            Articles about Continuous Integration & Delivery, automation, iOS code signing, DevOps, and awesome tech
          </p>
          <div
            className="subscription"
          >
            <form
              className="subscription-form"
              onSubmit={[Function]}
            >
              <div
                className="subscription-wrapper"
              >
                <div
                  className="email-container"
                >
                  <input
                    className=""
                    id="email-address"
                    name="email"
                    onChange={[Function]}
                    placeholder="Your email"
                    type="email"
                  />
                </div>
                <button
                  className="subscribe-button"
                  disabled={false}
                >
                  Subscribe
                </button>
              </div>
              <div
                className="subscription-wrapper success hidden"
              >
                <div
                  className="success-container"
                >
                  <div
                    className="check-wrapper"
                  >
                    <span
                      className="isvg loading"
                    />
                  </div>
                  <p />
                </div>
              </div>
            </form>
            <a
              className="feed"
              href="/atom"
              target="blank"
              value=" "
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
